uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/absence-tab.js

```diff
@@ -9,14 +9,19 @@
     }
   });
 
-  require([
-    'leave-absences/shared/config',
-    'leave-absences/absence-tab/app'
-  ], function () {
-    document.dispatchEvent(typeof window.CustomEvent == "function" ? new CustomEvent('absenceTabReady') : (function () {
-      var e = document.createEvent('Event');
-      e.initEvent('absenceTabReady', true, true);
-      return e;
-    })());
+  require(['leave-absences/shared/config'], function(__){
+    require(
+      [
+        'leave-absences/absence-tab/app'
```

Comment:
but if the problem is that `/app` has a dependency on `shared/config`, doesn't that mean that this can be easily solved by including it directly as a dependency when `/app` is defined?
```js
// leave-absences/absence-tab/app

define([
  'leave-absences/shared/config',
  // etc.
], function () {

});
```
this needs to be tested though, i don't remember now why shared/config was kept separate from /app (maybe something to do with tests setup?)